src/components/ui/form/calendar/components/CalendarTheme.tsx

```diff
@@ -26,27 +27,158 @@ export interface WeekDayTheme {
+  textProps?: TextPropsProvider<TUserData>;
+}
+
+interface DefaultWrapperColors {
+  selectedBorder: string;
+  selectedBackground: string;
+  rangeBorder: string;
+  rangeBackground: string;
+  todayBorder: string;
+  todayBackground: string;
+}
+
+const defaultWrapperStyleProvider = ({
+  selectedBackground,
+  todayBackground,
+  rangeBackground,
+  selectedBorder,
+  rangeBorder,
+  todayBorder,
+}: DefaultWrapperColors): WrapperStyleProvider<{}> => (
+  dayState,
+  day,
+  week,
+  month,
+) => {
+  const backgroundColor = dayHighlightSelect(
+    dayState,
+    ['selected', 'today', 'range', day.month === month.monthInYear],
+    [selectedBackground, todayBackground, rangeBackground, defaultColors.white],
+    'transparent',
+  );
+
+  if (day.month === month.monthInYear) {
+    return {
+      backgroundColor,
+      borderWidth: '1px',
+      borderColor: dayHighlightSelect(
+        dayState,
+        ['selected', 'range', 'today'],
+        [selectedBorder, rangeBorder, todayBorder],
+        defaultColors.separatorLight,
+      ),
+      borderCollapse: 'collapse',
+      borderStyle:
+        dayHasHighlight(dayState, 'selected') ||
+        dayHasHighlight(dayState, 'range') ||
+        dayHasHighlight(dayState, 'today')
+          ? 'double'
+          : 'solid',
+      boxSizing: 'border-box',
+      overflow: 'hidden',
+    };
+  }
+  return undefined;
+};
+
+interface DefaultTextColors {
+  disabledColor: string;
+  inOtherMonthColor: string;
+  selectedColor: string;
 }
 
+const defaultTextPropsProvider = ({
+  selectedColor,
+  disabledColor,
+  inOtherMonthColor,
+}: DefaultTextColors): TextPropsProvider<{}> => {
+  return (dayState, day, week, month) => {
+    const isOtherMonth = day.month !== month.monthInYear;
+    const color = dayHighlightSelect(
+      dayState,
+      [isOtherMonth, 'selected', 'disabled'],
+      [inOtherMonthColor, selectedColor, disabledColor],
+    );
+    return {
+      color,
+    };
+  };
+};
+
 export const defaultCalendarTheme: CalendarTheme = {
-  width: '50px',
-  height: '50px',
+  width: '40px',
+  height: '40px',
   WeekNumber: {
-    backgroundColor: '#F9F7F4',
+    backgroundColor: 'transparent',
+  },
+  WeekDay: {
+    textColor: defaultColors.separator,
+  },
+  CalendarDay: {
+    wrapperStyle: defaultWrapperStyleProvider({
+      selectedBorder: defaultColors.inputBorderFocused,
+      selectedBackground: defaultColors.inputBorderFocused,
+      rangeBorder: defaultColors.inputBorderFocusedAlt,
+      rangeBackground: defaultColors.inputBorderFocusedLight,
+      todayBorder: defaultColors.highlightBoxBorder,
+      todayBackground: defaultColors.highlightBoxBg,
+    }),
+    textProps: defaultTextPropsProvider({
+      selectedColor: defaultColors.white,
+      disabledColor: defaultColors.disabledText,
+      inOtherMonthColor: 'transparent',
+    }),
+  },
+  CalendarMonth: {},
+};
+
+export const extranetCalendarTheme: CalendarTheme = {
```

Comment:
We should move this constant to the extranet project instead?

Reply:
I'll leave it, maybe move it in another PR?